gitlab/webstandard.sh

```diff
@@ -80,6 +80,10 @@ if [ "$TEST" = "w3cval" ]; then
     rm -f localhost/domjudge/css/select2-bootstrap*.css*
     rm -f localhost/domjudge/css/dataTables*.css*
     rm -f localhost/domjudge/jury/config/check/phpinfo*
+    # We need any value for bs-gutter-x to calculate with in CSS
```

Comment:
I don't understand what is happening here. Do you remove this from the CSS file to avoid it being flagged as non-standard? Is there a risk that this doesn't work in "all" browsers then?

Reply:
Given that there are at this point only 2 browsers and it works there I think we're fine, also bootstrap already uses CSS variables so we would have problems somewhere else also.

The main problem is that W3C has the CSS standard where this is a very strong candidate (not sure how this works for them to make this final) but the validator doesn't detect this properly. There are a lot of threads about this that this is more an issue with the W3C validator than with support for this feature.

Reply:
Did you test this also in the mobile view?